docs/notebooks/thomson.ipynb

```diff
@@ -325,6 +316,11 @@
   }
  ],
  "metadata": {
+  "jupytext": {
+   "cell_metadata_filter": "incorrectly_encoded_metadata,-all",
+   "main_language": "python",
+   "notebook_metadata_filter": "-all"
+  },
```

Comment:
There's also this `jupytext` metadata that found it's way in.  Don't think it's a big deal though.

Reply:
Yup.